pkg/apis/core/v1alpha1/kubernetesdiscovery_types.go

```diff
@@ -281,6 +281,23 @@ type Pod struct {
 	Status string `json:"status" protobuf:"bytes,12,opt,name=status"`
 	// Errors are aggregated error messages for the Pod and its containers.
 	Errors []string `json:"errors" protobuf:"bytes,13,rep,name=errors"`
+
+	// Direct owner of this pod, if available.
+	Owner *PodOwner `json:"owner,omitempty" protobuf:"bytes,16,opt,name=owner"`
+}
+
+// PodOwner contains information of the direct owner of the
+// pod, if available.
+//
+// Tools that need to select a most relevant pod or set of pods
+// can use this info to group pods by owner.
+type PodOwner struct {
```

Comment:
Do we need/want any sort of GVK type metadata here?

Reply:
added APIVersion and Kind!